system/jlib/jtrace.cpp

```diff
@@ -67,6 +67,49 @@ namespace opentel_trace = opentelemetry::trace;
 
 using namespace ln_uid;
 
+class MeteredExporter final : public opentelemetry::sdk::trace::SpanExporter {
+private:
+    std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> exporter_;
+    StringBuffer exporterIdentifier;
+public:
+    MeteredExporter() = delete;
+    MeteredExporter(std::unique_ptr<opentelemetry::sdk::trace::SpanExporter> && exporter, const char * exporterIdentifier_) : exporter_(std::move(exporter))
+    {
+        exporterIdentifier.set(exporterIdentifier_);
+    }
+
+    opentelemetry::sdk::common::ExportResult Export(const nostd::span<std::unique_ptr<opentelemetry::sdk::trace::Recordable> > & spans) noexcept override
+    {
+        auto start = std::chrono::high_resolution_clock::now();
+        opentelemetry::v1::sdk::common::ExportResult result = exporter_->Export(spans);
+        auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::high_resolution_clock::now() - start);
+
+        //mimicking metric jlog output for now.
```

Comment:
Hardcoded metrics jlog output for now, could expose these values as histogram metric in future